openslides_backend/services/datastore/read_adapter.py

```diff
@@ -0,0 +1,160 @@
+from collections import defaultdict
+from typing import Any, ContextManager
+
+from datastore.reader.core import GetManyRequest, GetManyRequestPart
+from datastore.shared.di import injector
+from datastore.shared.postgresql_backend.connection_handler import ConnectionHandler
+from datastore.shared.postgresql_backend.sql_query_helper import SqlQueryHelper
+from datastore.shared.typing import Collection, Id
+
+Model = dict[str, Any]
+
+
+class ReadAdapter:
+    connection: (
+        ConnectionHandler  # TODO use PgConnectionHandlerService from datastore ?
```

Comment:
This theme we should talk about with @jsangmeister. Perhaps we could easily use that from datastore, but the  process/thread behavior is different.

Reply:
I think I am currently getting that connection handler anyway. As far as I understand it the datastore injector that I use to initialize this variable in ln 20 delivers the `PgConnectionHandlerService` when the ConnectionHandler is injected (as per [this file](https://github.com/OpenSlides/openslides-datastore-service/blob/main/datastore/shared/postgresql_backend/__init__.py)).